### PR TITLE
Docstring changes to reduce numpydoc validation issues, helps with #178

### PR DESCRIPTION
**Pull request by devendragovil**
Making some changes to docstrings. 

Note: I am not an expert in this part of the codebase and have changed docstrings based on my understanding.

### CODE REVIEW

File: toponetx/classes/cell_complex.py [hunk 1]
```diff
@@ -314,16 +366,30 @@ def __getitem__(self, node):
...
+        ----------
+        cell : tuple | list | Cell
+            The cell to insert.
+        **attr
+            The attributes of the cell to insert.
+
+        Returns
+        -------
+        None
+            Nothing is returned by this method.
```
Comment (ffl096) [inline toponetx/classes/cell_complex.py:391]: This should be removed.

File: toponetx/classes/cell_complex.py [hunk 2]
```diff
@@ -342,7 +408,20 @@ def _insert_cell(self, cell: tuple | list | Cell, **attr):
...
+        ----------
+        cell : tuple | list | Cell
+            The cell to delete.
+        key : Hashable
+            The key of the cell to delete.
+
+        Returns
+        -------
+        None
+            Nothing is returned by this method.
```
Comment (ffl096) [inline toponetx/classes/cell_complex.py:423]: This should be removed.

File: toponetx/classes/cell_complex.py (hunk 1 above)
Comment (devendragovil) [reply toponetx/classes/cell_complex.py:391]: Done!

File: toponetx/classes/cell_complex.py (hunk 2 above)
Comment (devendragovil) [reply toponetx/classes/cell_complex.py:423]: Done!

File: toponetx/classes/complex.py [hunk 3]
```diff
@@ -233,11 +265,18 @@ def __contains__(self, item: Any) -> bool:
         Returns
         -------
         bool
+            Returns `True` if the complex contains the item else `False`.
         """
 
     @abc.abstractmethod
     def __getitem__(self, key):
```
[the remark this answers is not in the document]
Comment (devendragovil) [reply toponetx/classes/complex.py:272]: Done!

**Review by ffl096 — APPROVED** [review]

**Pull request merged**